### ISSUE TITLE
Error on build/compile - Inavlid us of incomplete type 'class QAbstractItemView'

### ISSUE THREAD

**Issue by SeanDoh** [elieserdejesus/JamTaba#1139]
Hi All, amateur developer here hoping to contribute to the project!  I'm trying to compile the code per the readme to just poke around and get a feel for things, but and getting the following error:
![image](https://user-images.githubusercontent.com/1256271/47878571-02e3a700-ddf5-11e8-8dfd-1a64ab75fc57.png)

I've done some googling and searched the issues here on github but nothing relevant that I could see.
I don't know if I am missing a QT library or something?  I've never used QT before this.  I'm on windows 7 with Qt creator with 4.7.2 based on Qt 5.11.2.  Thanks for your time and any help!

Also, side question, is this the best place to ask any development question about jamtaba?  Are there any other developer forums?

**Comment by elieserdejesus** [comment]
Hi @SeanDoh, welcome!

> Also, side question, is this the best place to ask any development question about jamtaba? Are there any other developer forums?
This is right place to ask about JamTaba development.

I see in yout picture you are using Qt compiled to *MingW*. We dropped the support for MingW sometime ago, I think the problem was related with some 3rd part library compatibility issue.

All JamTaba devs are using MSVC (**I'm using MSVC 2013 express edition**, it's small and free), not MingW.

**To play with JamTaba code you must download**:
1 - MSVC (I recommend MSVC 2013 to avoid version issues) 
2 - Qt compiled for MSVC (your Qt version was compiled with MingW).

**Comment by SeanDoh** [comment]
Hi @elieserdejesus, thanks for the reply!  I've successfully got it compiled and running!  Much appreciated!

I did run into one issue with VST SDK versioning.  With the latest release, I got a build error 'cannot open file aeffectx.h'  I searched for the file on my computer and happened to have an older version of VST SDK (folder is called vstsdk3610_11_06_2018_build_37) that has the file and I was able to build and run.
